content/ko/docs/community/related.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "관련 프로젝트와 문서"
+description: "커뮤니티에서 제공하는 서드파티 도구, 플러그인 및 문서"
+weight: 3
+---
+
+헬름 커뮤니티는 헬름에 대한 많은 추가 도구, 플러그인 및 문서를 만들었습니다. 우리는
+이러한 프로젝트에 대해 듣고 싶습니다.
+
+이 목록에 추가하고 싶은 것이 있으면 
+[이슈](https://github.com/helm/helm-www/issues)나 [풀 
+리퀘스트(PR)](https://github.com/helm/helm-www/pulls) 할 수 있습니다.
```

Comment:
Instead of "pull request", the auto-translate thinks you're saying "full request". Does this convey the correct idea in Korean? Thanks.

Reply:
(I see https://github.com/helm/helm-www/pull/799#discussion_r487613943 but want to make sure it's correct here as well.)

Reply:
Merging because it's probably the same issue as in the other PR - but if it's not, please correct! Thanks.

Reply:
@bridgetkromhout Thanks you :D Both are same case.